dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -366,6 +374,14 @@ breeze build-docs --clean-build \
   ...
 ```
 
+You can also use shorthand names as arguments instead of using the full names
+for airflow providers. Example:
+
+```shell script
+cd "${AIRFLOW_REPO_ROOT}"
+breeze build-docs package1 package2 --clean-build
```

Comment:
How does this work for building `apache-airflow-providers` ?

Not that now you can remove the build_provider_documentation.sh part from the guide. Is this script being used somewhere? If not I guess we can remove it

Reply:
> How does this work for building apache-airflow-providers ?
I do not think there is a special rule for this. Might need to add one? We can follow the instructions above for this too, no need of any args

Yeah only here. I think we can safely remove it.

Reply:
> How does this work for building apache-airflow-providers ?
I do not think there is a special rule for this. Might need to add one? We can follow the instructions above for this too, no need of any args

>Not that now you can remove the build_provider_documentation.sh part from the guide. Is this script being used somewhere? If not I guess we can remove it
Yeah only here. I think we can safely remove it.

Reply:
But there is an issue: the script runs the add back references command as a part of it. 
```
breeze release-management add-back-references "${@}"
```

Reply:
cc @potiuk can we remove the script safely and document that elsewhere?

Reply:
Yes. We should make it as part of the follow-up. (@eladkal - it's backwards compatible so old ways will continue to work).

This could be done together with the changes I described for the CI. 

However - this is not THE RIGHT script/command :) 

This is `build-docs` command and the script is `build_provider_docuemntation.sh`. There is yet ANOTHER change to make in similar way - i.e. same change in `publish-docs` command. And only then we will be able to replace `publish_provider_documentation.sh`

Reply:
What I meant is that now we don't need the `build_provider_documentation.sh` any more so it can be removed.

I rarely used this script for released

> But there is an issue: the script runs the add back references command as a part of it.

there is no issue... the script handled both doc build and add back referenced but when I didn't use the script I simply ran both commands. I actually prefer it that way. it gives better control over the process.

Reply:
> there is no issue... the script handled both doc build and add back referenced but when I didn't use the script I simply ran both commands. I actually prefer it that way. it gives better control over the process.

(just to clarify - this is a different script :) ... it was about publishing, not building :)

> What I meant is that now we don't need the build_provider_documentation.sh any more so it can be removed.

Correct. But I want to really do that in the follow-up when we also switch  the CI to use it, to separte "breeze" changes with "using breeze" changes.